engine/engine/urls.py

```diff
@@ -67,6 +67,11 @@
         path("api/internal/v1/", include("apps.oss_installation.urls", namespace="oss_installation")),
     ]
 
+if settings.PHONE_PROVIDER == "zvonok":
```

Comment:
It is needed to check live_settings.PHONE_PROVIDER here. If it won't work ( I'm not remember if apps are loaded before urls, you can simply enable zvonok urls if settings.IS_OPEN_SOURCE (line 65)